### PR TITLE
Update theme for home page

### PR DESCRIPTION
**Pull request by uy-seng**
- Update themed homepage

### CODE REVIEW

File: components/homeComponents/HomeSponsors.tsx [hunk 1]
```diff
@@ -10,20 +18,28 @@ export default function HomeSponsors(props: { sponsorCard: Sponsor[] }) {
...
-            className="text-complementary font-bold md:text-4xl text-2xl my-4 text-center"
-          >
-            Our Sponsors
+      <section className="relative pt-[10rem] bg-[#DAC397]">
+        {/* TODO: will update styling better once get more assets and finalized content */}
+        <div>
+          <div className="text-center text-3xl text-white">
+            <h1 className="uppercase">see you there!</h1>
+          </div>
+          <div className="flex flex-wrap w-full items-end justify-center">
```
Comment (DedsecKnight) [inline components/homeComponents/HomeSponsors.tsx:27]: Can we do something like `align-center` to make them align with each other vertically?

File: hackportal.config.ts [hunk 2]
```diff
@@ -605,7 +605,7 @@ const setInitialValues = (obj) => {
 export const formInitialValues = getInitialValues();
 
 export const config = {
-  targetDate: '2024-11-04T00:00:00Z',
+  targetDate: '2024-11-16T00:00:00Z',
```
Comment (DedsecKnight) [inline hackportal.config.ts:608]: Make sure this matches our timezone. If it already matches, feel free to resolve this comment.

**Review by DedsecKnight — CHANGES_REQUESTED** [review]
I believe this is a bug
Shot-2024-07-28-204543

**Comment by raywa04** [comment]
> I believe this is a bug
> Shot-2024-07-28-204543

can you recheck on ur side if the bug is still happening pls

**Pull request merged**